app/src/main/res/layout/dialog_report.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"-->
+    <!--xmlns:app="http://schemas.android.com/apk/res-auto"-->
```

Comment:
Could we remove all the comments here or do you have plans to work on that again in the future and think that it is good to leave them here?

Reply:
It's still WIP, I plan to tweak this later on.